frontend/src/components/student/StudentDashboardHeader.tsx

```diff
@@ -1,23 +1,27 @@
 import React from "react";
-import { Text, VStack } from "@chakra-ui/react";
+import { Spacer, Text, VStack } from "@chakra-ui/react";
```

Comment:
can we center align the header if classroomName is empty?
<img width="1436" alt="image" src="https://github.com/uwblueprint/jump-math/assets/69697180/275db2d0-b88a-4684-8272-3d0582d31d96">

Reply:
yes ma'am